src/javascript/app_2/Modules/Statement/Components/card_list_desktop.jsx

```diff
@@ -20,6 +21,7 @@ const CardListDesktop = ({
 export default connect(
     ({ modules }) => ({
         data            : modules.statement.data,
+
         columns         : [
             { title: localize('Daaaate'),          data_index: 'date' },
```

Comment:
Daaaate? 😄

Reply:
Good catch, though it's already fixed in the [next commit](https://github.com/binary-com/binary-static/pull/4060/commits/ece077072cbcc3c10f6bb35f145cfee4279ad84e#diff-8ce3a23410ba79a9192de9ebc6c85b75R30). 😄